CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Update SNAP JSON file structure and add participation percentages. [#48](https://github.com/policy-design-lab/pdl-api/issues/48)
+- Make summary and all programs directly served from json file. [#65](https://github.com/policy-design-lab/pdl-api/issues/65)
```

Comment:
Please move this to the unreleased section. The rest of the changes look good. Thanks.

Reply:
updated